src/Xamarin.Android.Build.Tasks/Tasks/AndroidUpdateResDir.cs

```diff
@@ -96,8 +97,8 @@ public override bool Execute ()
 					if (string.IsNullOrEmpty (rel)) {
 						rel = item.GetMetadata ("Identity");
 						if (!string.IsNullOrEmpty (ProjectDir)) {
-							var fullRelPath = Path.GetFullPath (rel);
-							var fullProjectPath = Path.GetFullPath (ProjectDir);
+							var fullRelPath = Path.GetFullPath (rel).Normalize (NormalizationForm.FormC);
```

Comment:
The normalization here *is* required; the result of this normalization isn't used for subsequent filesystem reads/written to a file, but is instead used for relative path computation, and thus should be fine.  I think.